lepidopter-fh/etc/logrotate.d/ooniprobe

```diff
@@ -1,10 +1,9 @@
 /var/log/ooni/*.log {
-    weekly
     missingok
-    rotate 2
+    rotate 1
     compress
     delaycompress
     copytruncate
-    maxsize 10M
+    maxsize 1M
     notifempty
```

Comment:
Until the ooniprobe logging situation is being resolved we can use this.
You should consider that  `/var/log/ooni/cronjobs.log` need to be processed by logrotate as well.